Bio/SubsMat/__init__.py

```diff
@@ -351,14 +370,28 @@ def __sub__(self, other):
     def __mul__(self, other):
         """Matrix multiplication.
 
-        Returns a matrix for which each entry is the multiplication product of
-        the two matrices passed.
+        Returns a new matrix created by multiplying each element by other (if
+        other is scalar), or by performing element-wise multiplication of the
+        two matrices (if other is a matrix of the same size).
```

Comment:
Looking at this docstring, the one-line summary is misleading: "Matrix multiplication."

How about "Element-wise matrix multiplication." instead?

See also new ``@`` matrix multiplication operator in Python 3.5+ which we'll be able to start using early 2020 once we drop Python 2.7 (since we've already dropped Python 3.0 - 3.4 support).